bridge-impl/src/main/java/com/liferay/faces/bridge/context/internal/RenderRedirectWriterImpl.java

```diff
@@ -108,88 +108,118 @@ public Writer getWrapped() {
 		void invoke() throws IOException;
 	}
 
-	protected class CBufOffLenOutputOperation implements OutputOperation {
+	private static class CBufOffLenOutputOperation implements OutputOperation {
 
 		private char[] cbuf;
 		private int off;
 		private int len;
+		private Writer myWriter;
 
-		public CBufOffLenOutputOperation(char[] cbuf, int off, int len) {
+		public CBufOffLenOutputOperation(Writer writer, char[] cbuf, int off, int len) {
 			this.cbuf = cbuf.clone();
 			this.off = off;
 			this.len = len;
+			this.myWriter = writer;
 		}
+		
```

Comment:
@pcwhite Please remove the commented-out code block, squash your commits, and push to your branch again.